sympy/printing/mathml.py

```diff
@@ -507,11 +506,26 @@ def mathml_tag(self, e):
             'StrictLessThan': '<',
         }
 
+        def mul_symbol_selection():
+            if self._settings["mul_symbol"] == None or self._settings["mul_symbol"] == 'None':
+                return '&InvisibleTimes;'
+            elif self._settings["mul_symbol"] == 'times':
+                return '&#xD7;'
+            elif self._settings["mul_symbol"] == 'dot':
+                return '&#xB7;'
+            elif self._settings["mul_symbol"] == 'ldot':
+                return '&#x2024;'
+            elif not isinstance(self._settings["mul_symbol"], str):
```

Comment:
This is where string_types should be instead of str. The reason is that depending on Python version, str is sometimes not enough to cover strings.